include/z64animation.h

```diff
@@ -88,12 +88,30 @@ typedef enum {
     /*  1 */ ANIMTAPER_ACCEL
 } AnimationTapers;
 
-#define ANIM_FLAG_0 (1 << 0) // (no effect outside of player) Related to scaling an animation from/to child/adult
+// This flag seems like it was intended to be paired with `ANIM_FLAG_UPDATE_Y` to control 
+// XZ movement based on the current animation.
+// However, this flag is not checked by the Skelanime system. It will not toggle XZ movement.
```

Comment:
Is XZ movement is always applied (unless ANIM_FLAG_NO_MOVE is set)? Maybe worth stating that here, right now it sort of implies XZ movement is never applied